build.sh

```
@@ -14,37 +15,64 @@ MOD=$(basename $PWD)
 solutionPath="Source/${MOD}.sln"
 
 # Define an array of configurations
-configurations=("Release v1.2" "Release v1.3" "Release v1.4" "Release v1.5")
+configurations=("v1.2" "v1.3" "v1.4" "v1.5" "v1.6")
+
+dotnet restore "$solutionPath"
+
+function sync_mod() {
+    # Copy over the mod directory.
+    rsync -a ${MOD} /rimworld/1.2/Mods/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)